extra-wallpaper/aosc-community-wallpapers-2020/spec

```diff
@@ -0,0 +1,4 @@
+VER=1.0.0
+GITTAG=0.0.1-pre2
```

Comment:
You should consider a new versioning scheme for WallColle. Two versions for one such simple package is unnecessarily complicating things.